LawnCare/frontend/app/src/main/java/com/redflag/lawncare/book/BookNowFragment.java

```diff
@@ -96,7 +96,7 @@ public void onFailure(@NonNull Exception e) {
 
     private boolean checkForError(EditText text, boolean errorState) {
         if("".equals(text.getText().toString())){
-            text.setError(getResources().getString(R.string.err_cant_be_empty));
+            text.setError(getString(R.string.err_cant_be_empty));
```

Comment:
this has already been fixed, hasnt it?

Reply:
oh weird. my ver had the actual string in there but the string.xml had your strings in there so i just switched them in.